lib/ssl/src/ssl_crl_cache.erl

```diff
@@ -143,8 +143,12 @@ delete({der, CRLs}) ->
 
 delete(URI) ->
     case uri_string:normalize(URI, [return_map]) of
-	#{scheme := "http", path := Path} ->
-	    ssl_manager:delete_crls(string:trim(Path, leading, "/"));
+	#{scheme := "http",
+          host := Host,
+          path := Path} = Map ->
+            Port = maps:get(port, Map, 80),
+            Key = Host ++ ":" ++ integer_to_list(Port) ++ Path,
```

Comment:
maybe have a helper function wrapping
`Host ++ ":" ++ integer_to_list(Port) ++ Path`

so same procedure is always followed and it is harder to make a mistake by altering to few places ...